staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go

```diff
@@ -85,7 +85,9 @@ func WithLogging(handler http.Handler, pred StacktracePred) http.Handler {
 		rl := newLogged(req, w).StacktraceWhen(pred)
 		req = req.WithContext(context.WithValue(ctx, respLoggerContextKey, rl))
 
-		defer func() { klog.V(3).InfoS("Received HTTP request", rl.LogArgs()...) }()
+		if klog.V(3).Enabled() {
+			defer func() {klog.InfoS("HTTP", rl.LogArgs()...)}()
```

Comment:
```
defer klog.InfoS("HTTP", rl.LogArgs()...)
```
?

Reply:
that would change the moment of running LogArgs, is that indifferent?

Reply:
LogArgs measures time for request, we cannot use 
```
defer klog.InfoS("HTTP", rl.LogArgs()...)
```
as this would evaluate ` rl.LogArgs()` immidietly